### PR TITLE
Issue 2173

### PR DESCRIPTION
**Pull request by Lysette**
Shows which submission is collected in repo browser.

### CODE REVIEW

File: app/views/submissions/repo_browser.html.erb [hunk 1]
```diff
@@ -62,6 +62,9 @@
         (<span id='current_revision_timestamp_display'>
            <%= l(@revision_timestamp, format: :long_date) %>
         </span>)
+        <br>
+        <%= raw(t('browse_submissions.currently_collected')) %>
```
Comment (david-yz-liu) [inline app/views/submissions/repo_browser.html.erb]: You shouldn't need to use `raw` here

File: app/views/submissions/repo_browser.html.erb [hunk 2]
```diff
@@ -62,6 +62,9 @@
         (<span id='current_revision_timestamp_display'>
            <%= l(@revision_timestamp, format: :long_date) %>
         </span>)
+        <br>
+        <%= raw(t('browse_submissions.currently_collected')) %> 
+        <%= @grouping.is_collected ? "#" + raw(@grouping.submissions.last.revision_number) + " ( " + l((@revisions_history.select {|rev| rev[:num] == @grouping.submissions.last.revision_number }).first[:date], format: :long_date) + " )" : "none" %>
```
Comment (david-yz-liu) [inline app/views/submissions/repo_browser.html.erb]: This line is too long and needs to be split up.

File: app/views/submissions/repo_browser.html.erb [hunk 3]
```diff
@@ -62,6 +62,11 @@
         (<span id='current_revision_timestamp_display'>
            <%= l(@revision_timestamp, format: :long_date) %>
         </span>)
+        <br>
+        <%= t('browse_submissions.currently_collected') %> 
+        <%= @grouping.is_collected ? "#" + raw(@grouping.submissions.last.revision_number) +
```
Comment (david-yz-liu) [inline app/views/submissions/repo_browser.html.erb]: Hmm okay, after looking at this more closely, I have three comments:

1. I'd rather use an `if` here than the ternary operator. 
    ```
      <% if @grouping.is_collected%>
        <% ="#" + @grouping.submissions.last.revision_nubmer %>
      ...
    ```
2. Probably don't need raw for the revision number either
3. I *think* that you can do something like `@grouping.submissions.last.revision_timestamp` to get the timestamp for the submission. I was just looking at that today.

Comment (Lysette) [inline app/views/submissions/repo_browser.html.erb]: <code>@grouping.submissions.last.revision_timestamp</code> returns incorrect time which is why I had to use <code>@revisions_history</code>.

**Pull request merged**